source/settings/pigeonhole/duplicate.rst

```diff
@@ -0,0 +1,27 @@
+=====================================
+Pigeonhole Sieve: Duplicate Extension
+=====================================
+
+.. _plugin-sieve-setting-sieve_duplicate_default_period:
+
+``sieve_duplicate_default_period``
+----------------------------------
+
+ - Default: ``14d``
+ - Value: :ref:`time`
+
+Default period after which tracked values are purged from the duplicate tracking database.
+The period is specified in s(econds), unless followed by a d(ay), h(our) or m(inute) specifier character. 
+
+.. _plugin-sieve-setting-sieve_duplicate_max_period:
+
+``sieve_duplicate_max_period``
+------------------------------
+
+ - Default: ``7d``
+ - Value: :ref:`time`
+
+Maximum period after which tracked values are purged from the duplicate tracking database.
+The period is specified in s(econds), unless followed by a d(ay), h(our) or m(inute) specifier character.
+
+See also :ref:`pigeonhole_extension_duplicate`.
```

Comment:
Not all plugin/extension settings pages have this reference.

Also, this reference should likely be at the start of the page.

Reply:
why at start of page?

Reply:
Should probably have a short introduction there with a link towards more information.

Reply:
And at the end of the page the link seems to belong to the last setting described.

Reply:
it's now using `.. seealso::` on top of page.